### ISSUE TITLE
Multiple Outputs?

### ISSUE THREAD

**Issue by quidmonkey** [grame-cncm/faustide#45]
Is it possible to send multiple outputs with the Faust Web IDE? I use a Focusrite Scarlett 18i20, which features up to 10 outputs. I would like to send a quad output, and not just a stereo output. I have tried various tests, but none seem to work. Is this possible?

Thanks.

**Comment by quidmonkey** [comment]
Here's some context to what I'm asking. I'm attempting to run this [simple Faust quad test](https://faustide.grame.fr/?autorun=1&voices=0&name=quadtest&inline=aW1wb3J0KCJzdGRmYXVzdC5saWIiKTsNCg0Kdm9sdW1lID0gaHNsaWRlcigiVm9sdW1lW3N0eWxlOmtub2JdIiwgMTAsIDAsIDEwMCwgMSkgLyAxMDAgOiBzaS5zbW9vOw0KDQovLyBtYWpvciBzZXZlbnRoDQp0b25pYyA9IDIyMDsNCnRoaXJkID0gdG9uaWMgKiA1IC8gNDsNCmZpZnRoID0gdG9uaWMgKiAzIC8gMjsNCnNldmVudGggPSB0b25pYyAqIDE1IC8gODsNCg0KLy8gYXBwbHkgZ2Fpbg0KbGVmdF9mcm9udCA9IG9zLnNhd3Rvb3RoKHRvbmljKSAqIHZvbHVtZTsNCnJpZ2h0X2Zyb250ID0gb3Muc2F3dG9vdGgodGhpcmQpICogdm9sdW1lOw0KbGVmdF9iYWNrID0gb3Muc2F3dG9vdGgoZmlmdGgpICogdm9sdW1lOw0KcmlnaHRfYmFjayA9IG9zLnNhd3Rvb3RoKHNldmVudGgpICogdm9sdW1lOw0KDQpwcm9jZXNzID0gbGVmdF9mcm9udCwgcmlnaHRfZnJvbnQsIGxlZnRfYmFjaywgcmlnaHRfYmFjazsNCg%3D%3D) on Windows 10 using Chrome (latest version). Here's what I've done to test this issue:

1. I have checked my Windows 10 sound settings and verified that I have my Focusrite Speakers setup to be quadraphonic.
2. I tested my speaker setup to ensure that I can get quadraphonic sound through Windows 10 using other sources.
3. I opened Chrome, created a `new AudioContext()` and verified that the `.destination.maxChannels = 4`. By default, `.destination.channels = 2`. I have set `.destination.channels = 4` with no issue, and can use the `AudioContext` in quadraphonic.

Despite all this, I see no way to map the Faust Web IDE to support anything other than stereo output? Perhaps this goes under an enhancement, but I would love to see the Faust Web IDE feature an Audio Output dropdown (just below the Audio Input) that would allow you to map the number of outputs up to the maxChannels as allowed by the AudioContext.
